skill_markII_audio_receiver/__init__.py

```diff
@@ -48,6 +48,15 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.renaming_airplay = False
         self.renaming_spotify = False
+        self.initialize()
+
+    def initialize(self):
```

Comment:
I believe everything here can just be appended to `__init__` to resolve the test failures. As of `ovos-workshop` 0.0.12 init should handle bus init in the `super()` call